astropy/samp/__init__.py

```diff
@@ -27,12 +27,13 @@ class Conf(_config.ConfigNamespace):
 
     use_internet = _config.ConfigItem(
         True,
-        "Whether to allow `astropy.samp` to use "
-        "the internet, if available.",
-        aliases=['astropy.samp.utils.use_internet'])
+        "Whether to allow `astropy.samp` to use " "the internet, if available.",
```

Comment:
We should fix this string concatenation (either here or in follow-up across package)

Reply:
Fixed by the `--preview` flag!